vue.config.js

```diff
@@ -27,8 +27,8 @@ module.exports = {
         'israeli-bank-scrapers-core'
       ],
       builderOptions: {
-        productName: 'israeli-bank-scrapers-desktop',
-        appId: 'com.electron.israeli-bank-scrapers-desktop',
+        productName: globals.APP_NAME,
+        appId: 'com.electron.' + globals.APP_NAME,
```

Comment:
This is actually not good.

You can see in globals, we not just creating an object with keys and values like this:

```js
const clobals = {
  APP_NAME: "Hiuvi"
}
```

Instead, we using `JSON.stringify`, so the value is a string with quates. Becuase when it replace it in the code, it should put the quates also. It confusing, but at the end you get `appId: 'com.electron."Hiuvi"`

This is why the build failed.

I see some options here:

1. Just set the value here.
2. Export the `APP_NAME` itselfs from the `globals.config.js` file, alongside the clobals (I'm not sure how to do that).
3. Keep the `APP_NAME` value in the `package.json` file, with custom field, see [here](https://stackoverflow.com/a/27232456/839513).

Reply:
See this line in the build: https://github.com/brafdlog/budget-tracking/pull/192/checks#step:8:92

Reply:
Maybe I'll just use the name from the package.json.
It probably also makes more sense to use the name without uppercase.
The disadvantage is importing anther thing.

L2: `import {name as appName} from './package.json';`

L30: `builderOptions: {`
          `productName: appName,`
          `appId: 'com.electron.' + appName,`

Reply:
I'm not sure I understand the disadvantage. Do you mean that adding the `import` line is the disadvantage?

I think it is really OK.